package-lock.json

```diff
@@ -8416,7 +8416,7 @@
     },
     "diffable-html": {
       "version": "4.0.0",
-      "resolved": "https://registry.npmjs.org/diffable-html/-/diffable-html-4.0.0.tgz",
+      "resolved": "https://npm.morph.int.tools.bbc.co.uk/diffable-html/-/diffable-html-4.0.0.tgz",
```

Comment:
this change should probably be removed

Reply:
yup, not sure why this is happening still :(

Reply:
have pushed the fix @jblaut